lib/ClangImporter/ClangImporter.cpp

```diff
@@ -720,14 +774,9 @@ ClangImporter::create(ASTContext &ctx,
   auto ppTracker = llvm::make_unique<BridgingPPTracker>(importer->Impl);
   clangPP.addPPCallbacks(std::move(ppTracker));
 
-  // If we're going to load a bridging PCH, we need to know this now in order to
-  // clobber clangPP's predefined-macros buffer, otherwise it will provide macro
-  // definitions that collide with those in the PCH. This is actually simulating
-  // something clang does when it attaches to a PCH ASTReader as well, it's just
-  // interleaving the steps in an unusual order.
-  if (llvm::sys::path::extension(importerOpts.BridgingHeader).endswith(
-        PCH_EXTENSION))
-    clangPP.setPredefines("");
+  // FIXME: This is missing implicit includes.
```

Comment:
Is this FIXME still true? This is about there being `-include` options passed via `-Xcc` and the importer not exposing the contents.